roles/migrid_logstash_gdp/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# only use logstash if the server is a SiF server. Here i know that from my inventory.yml
+- name: Import logstash task if needed
+  import_tasks:
+    file: logstash.yml
+  when: '"sif" in group_names'
```

Comment:
Why do not decide based on the `migrid_base_type` which is already existing and must used anyway?
I think a Migrid configuration should not make assumptions on the group names that users have. Eg. KU has different names for SIF server groups.